From f73ea1eb4cce66376cc1dd94b4a083ffb9eeb123 Mon Sep 17 00:00:00 2001
Message-Id: <f73ea1eb4cce66376cc1dd94b4a083ffb9eeb123.1649662062.git.geliang.tang@suse.com>
From: Martin KaFai Lau <kafai@fb.com>
Date: Fri, 2 Apr 2021 17:29:21 -0700
Subject: [PATCH] bpf: selftests: Specify CONFIG_DYNAMIC_FTRACE in the testing
 config
Git-commit: f73ea1eb4cce66376cc1dd94b4a083ffb9eeb123
Patch-mainline: v5.13-rc1
References: bsn#127
Modified-by-SEL: No


The tracing test and the recent kfunc call test require
CONFIG_DYNAMIC_FTRACE.  This patch adds it to the config file.

Signed-off-by: Martin KaFai Lau <kafai@fb.com>
Signed-off-by: Andrii Nakryiko <andrii@kernel.org>
Link: https://lore.kernel.org/bpf/20210403002921.3419721-1-kafai@fb.com
Signed-off-by: Geliang Tang <geliang.tang@suse.com>
---
 tools/testing/selftests/bpf/config | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/tools/testing/selftests/bpf/config b/tools/testing/selftests/bpf/config
index 37e1f303fc11..5192305159ec 100644
--- a/tools/testing/selftests/bpf/config
+++ b/tools/testing/selftests/bpf/config
@@ -44,3 +44,5 @@ CONFIG_SECURITYFS=y
 CONFIG_IMA_WRITE_POLICY=y
 CONFIG_IMA_READ_POLICY=y
 CONFIG_BLK_DEV_LOOP=y
+CONFIG_FUNCTION_TRACER=y
+CONFIG_DYNAMIC_FTRACE=y
-- 
2.34.1

